src/color.rs

```diff
@@ -1591,3 +1608,45 @@ where
 
     Ok((r1, r2, r3, alpha))
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn serialize_modern_components() {
```

Comment:
(Not convinced this test is particularly useful but...)

Reply:
What would you change to be convinced?